### ISSUE TITLE
accuary on UCF101 or Kinetics

### ISSUE THREAD

**Issue by Kathrine94** [hassony2/kinetics_i3d_pytorch#14]
Hello! I use your code to train UCF101 dataset, but loss does not go down.Can you show me ｈ-yperparameters when you train the model?Does this code training model achieve the effect of the paper?

**Comment by hassony2** [comment]
Hi @Kathrine94,

Sorry for the late response.
I successfully trained (or more precisely, finetuned) this network on other datasets  (not UCF101 though).
I used cross-entropy loss SGD  with:
- lr 0.001
- momentum 0.9
Hope this helps ! 
Best,

Yana
